tasks/markup-1.md

```diff
@@ -1,105 +1,73 @@
-**!оценка: 0-10**
+1. General:
+    - the name of the HTML file should be *index.html*;
+    - Create individual folders for images, fonts, styles. If all the files will be placed on the same level as 'index.html' file - **minus 10 points**;
+    - The markup width should be 1600px. Hence the layout will be tested on the 1600px screen-width or higher. The markup should be centered. If you are going to implement the task on the device with screen width lower then 1600px, the scrollbar will appear, and It's Ok(you can use zoom out)(**points - 0-10**);
 
-9. Форма должна быть оформлена в виде тега form и всего надлежащего внутри. Форма должна содержать базовую браузерную валидацию на Email, а поля Name, Email и Date должны быть required. В макете, учтите, ошибка в названии поля. У вас ее быть не должно.
-**!оценка: 0-10**
+2. Semantic layout:
+	 - HTML5 semantic tags should be used, such as header, nav, section, article, footer, form + fields, figure  (**points - 0-10**);
+	 - Class names should be reasonable and correct, avoid names such as .right-column, .third-form, .left, .big. use more meaningful ones - .heading, .form-container, .wrapper, .menu-column. Usage of the pseudo-classes are very welcome (:first-child, last-child, etc.)(**points - 0-10**);
+	 - Using id for styles will be considered as a mistake, **minus 5 points**
 
-10. Footer должен прибит к низу. Прибитие значит, что если убрать контент на странице, кроме футера, то футер будет все равно внизу.
-**!оценка: 0-5**
+3. The logo should be implemented correctly. Don't forget to use *h1* tag on the page(**points - 0-5**);
 
-11. На сайте динамики может не быть, кроме событий hover. К ним можно и нужно применить стилевое оформление на ваше усмотрение, например при наведении курсором на элементы верхенго меню.
-**!оценка: 0-5**
+4. Navigation should be implemented with a *nav* tag. Clickable parts of the menu should be *links*(**points - 0-5**);
 
-Header можно делать фиксированным при прокрутке на фоне градиента, но доп. баллов за это не предусмотрено.
+5. Pay your attention to the fact that the background should be a graphic pattern, not just plain white(**points - 0-5**);
 
-Использовать можно любую сеточную технологию. Можно использовать font-awesome для иконок.
+6. Fonts. You can find all the font for free (the links will be provided). The fonts should be imported properly. Don't forget about the font-weight, it also should be correct(**points - 0-10**);
 
-Препроцессоры и постпроцессоры использовать не обязательно. Отдавать на выходе файлы типа .scss, .less нельзя. Также нельзя использовать bootstrap или подобную библиотеку. Все должно быть сделано самостоятельно. штраф **минус 40 баллов**.
+7. Requirements for the blocks and columns:
+	 - The case with more text than in the markup should be considered - just copy to the column bigger text(the position of the column or layout shouldn't change, limitation by overflow)(**points - 0-10**)
```

Comment:
Consider the case with more text than in the markup - just add more text to the column and check if the position of the column or layout does not change (limitation by overflow)(**points - 0-10**)